extensions/wikia/Email/EmailImageHelper.class.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Email;
+
+/**
+ * Class ImageHelper
+ *
+ * @package Email
+ */
+class ImageHelper {
+	static public $icons = [
+		'Wikia',
+		'Comics',
+		'Movies',
+		'Lifestyle',
+		'Music',
+		'Books',
+		'TV',
+		'Twitter',
+		'Facebook',
+		'YouTube',
+	];
+
+	/**
+	 * Get URL, size and name information about icons created in the newsletter wikia for our emails
+	 *
+	 * @return array
+	 */
+	public static function getIconInfo() {
+		$info = [];
+
+		foreach ( self::$icons as $name ) {
+			$file   = \GlobalFile::newFromText( $name . '.gif', \Wikia::NEWSLETTER_WIKI_ID );
```

Comment:
```GlobalFile``` isn't cached in memcache - this means that every email will perform ~10 additional DB queries. What do you think about caching them?